CMSgov/easi-app
# [EASI-2955] Refactor TRB email recipient fields

# EASI-2955

## Changes and Description

- Refactored TRB email recipient fields to not add new recipients as attendees
- Created `useActionForm` hook and `ActionFormWrapper` component to standardize action forms
- Fixed field validation bug that prevented user from submitting form with one attendee selected



## How to test this change

- Test email recipients form on TRB actions
  - [Request Edits](http://localhost:3000/trb/82213b1c-4530-4954-af39-e04d101d1304/request/request-edits)
  - [Ready for Consult](http://localhost:3000/trb/82213b1c-4530-4954-af39-e04d101d1304/request/ready-for-consult)
  - [Schedule Consult](http://localhost:3000/trb/82213b1c-4530-4954-af39-e04d101d1304/request/schedule-consult)
  - [Close Request](http://localhost:3000/trb/82213b1c-4530-4954-af39-e04d101d1304/request/close-request)
  - [Reopen Request](http://localhost:3000/trb/82213b1c-4530-4954-af39-e04d101d1304/request/reopen-request)
- Confirm that form submits with only one recipient selected
- Confirm that recipients are not added as attendees



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
